### ISSUE TITLE
`git lfs status` file names are wrong for files with the same content

### ISSUE THREAD

**Issue by larsxschneider** [github/git-lfs#1530]
### Setup

```
git-lfs/1.4.1 (GitHub; windows amd64; go 1.7; git 2070e4c) 
```

### Steps to reproduce

```bash
$ echo "test" > test1.dat
$ echo "test" > test2.dat
$ git lfs track *.dat
Tracking test1.dat
Tracking test2.dat
$ git add .
$ git lfs status
On branch develop
Git LFS objects to be pushed to origin/develop:


Git LFS objects to be committed:

    test2.dat (5 B)
    test2.dat (5 B)

Git LFS objects not staged for commit:
```

`git lfs status` should print `test1.dat` instead of the first `test2.dat`

**Comment by technoweenie** [comment]
Thanks for the report. Added to a v1.5.0 milestone. If @ttaylorr or someone else can submit a patch, I can see getting this into a v1.4.x release, but I personally don't have the time now.

**Comment by ttaylorr** [comment]
Thanks for the ping. I can take a look at this after I finish some work I'm doing on the transferqueue. My hunch is that this is (somewhat) related to https://github.com/github/git-lfs/pull/1458.

**Comment by ttaylorr** [comment]
https://github.com/github/git-lfs/pull/1550.

**Comment by ttaylorr** [comment]
Fixed in #1550!
